gradle.properties.example

```diff
@@ -1,6 +1,7 @@
-signing.keyId=KEYID
-signing.password=KEY_PASSWORD
-signing.secretKeyRingFile=KEY_PATH
+#skipSigning=false
+#signing.keyId=KEYID
+#signing.password=KEY_PASSWORD
+#signing.secretKeyRingFile=KEY_PATH
 
-nexusUsername=SONATYPE_LOGIN
-nexusPassword=SONATYPE_PASSWORD
+#nexusUsername=SONATYPE_LOGIN
+#nexusPassword=SONATYPE_PASSWORD
```

Comment:
For some reason if you just copy build.properties.example to build.properties and leave broken entries for sign* properties the build will fail.

It works if sign* properties are not defined at all.